src/routes/index.svelte

```diff
@@ -49,43 +49,95 @@
   });
 </script>
 
+<style lang="scss">
+  .banner {
+    --banner-child-v-spacing: 2.5rem;
+  }
+
+  .banner--icons {
+    padding: 0.5rem 1rem;
+    gap: 0.25rem;
+    margin-bottom: var(--banner-child-v-spacing);
+  }
+
+  .banner--content {
+    > * + * {
+      margin-top: var(--banner-child-v-spacing);
+    }
+
+    .title-container * {
+      display: inline;
+    }
+
+    .title-container p {
+      filter: drop-shadow(0px 1px 2px #121416);
+      max-width: 50ch;
+    }
+
+    p.lead {
+      font-weight: 400;
+      max-width: 56ch;
+      margin-bottom: 0;
+    }
+
+    .title {
+      font-size: 2rem;
+    }
+  }
+
+  .btn-container {
+    text-transform: uppercase;
+  }
+
+  .footer {
+    margin-top: 2rem;
+
+    @media (max-width: 1000px) {
+      margin-top: 1rem;
+    }
+  }
+</style>
+
 <svelte:head>
   <title>Cal-Adapt</title>
 </svelte:head>
 
 <!-- Banner -->
 <section
-  class="banner overlay overlay-black overlay-40"
+  class="bleed banner overlay overlay-black overlay-40"
   style="background-image: url(/img/banners/yosemite_2000w.jpg);background-size:cover;"
 >
   <div class="bx--grid">
-    <div class="bx--row" style="padding: 0.5rem 1rem;">
+    <div class="bx--row banner--icons" aria-hidden="true">
```

Comment:
Do we want to add a single anchor link at the row level which goes to the Tool page instead?

Reply:
My gut feeling is that I question the necessity of making these icons go to the tools page because we already link to that page twice; once in the navigation and another in the cards. If they were to link to the tools page then I think what would be ideal would be that they apply the filter associated with the icon to the list of tools.